index.html

```diff
@@ -73,117 +75,155 @@ <h1>
+          <button
+            class="btn work-section__btn"
+            type="submit"
+            data-btn="#project2"
+          >
             See Project
           </button>
         </div>
       </div>
 
-      <div class="card3 card-work">
+      <div class="card3 card-work" id="project3">
         <div class="card-work__img">
-          <img src="./img/work3.svg" alt="Project-3 image" />
+          <img
+            src="./img/work3.svg"
+            alt="Project-3 image"
+            data-img="card-image"
+          />
         </div>
         <div class="card-work__info">
-          <h2>Tonic</h2>
-          <div class="frame2">
+          <h2 data-title="title">Tonic</h2>
+          <div class="frame2" data-frame="frame-list">
             <p>CANOPY</p>
             <ul>
               <li>Back End Dev</li>
               <li>2015</li>
             </ul>
           </div>
-          <p>
+          <p data-p="paragraph">
             A daily selection of privately personalized reads; no accounts or
             sign-ups required.
           </p>
-          <ul class="programing-lamguages">
+          <ul class="programing-lamguages" data-language="language-list">
             <li>html</li>
             <li>css</li>
             <li>javaScript</li>
           </ul>
-          <button class="btn work-section__btn" type="submit">
+          <button
+            class="btn work-section__btn"
+            type="submit"
+            data-btn="#project3"
+          >
             See Project
           </button>
         </div>
       </div>
 
-      <div class="card4 card-work work-reversed-row">
+      <div class="card4 card-work work-reversed-row" id="project4">
         <div class="card-work__img">
-          <img src="./img/work4.svg" alt="Project-4 image" />
+          <img
+            src="./img/work4.svg"
+            alt="Project-4 image"
+            data-img="card-image"
+          />
         </div>
         <div class="card-work__info">
-          <h2>Multi-Post Stories</h2>
-          <div class="frame2">
+          <h2 data-title="title">Multi-Post Stories</h2>
+          <div class="frame2" data-frame="frame-list">
             <p>CANOPY</p>
             <ul>
               <li>Back End Dev</li>
               <li>2015</li>
             </ul>
           </div>
-          <p>
+          <p data-p="paragraph">
             A daily selection of privately personalized reads; no accounts or
             sign-ups required.
           </p>
-          <ul class="programing-lamguages">
+          <ul class="programing-lamguages" data-language="language-list">
             <li>html</li>
             <li>css</li>
             <li>javaScript</li>
           </ul>
-          <button class="btn work-section__btn" type="submit">
+          <button
+            class="btn work-section__btn"
+            type="submit"
+            data-btn="#project4"
+          >
             See Project
           </button>
         </div>
       </div>
```

Comment:
- [ ] Kindly create the project section with JS, you have to remove it from HTML and create it dynamically using JS when the page loads. As mentioned in the requirements `You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.`